libnavigation-core/src/main/java/com/mapbox/navigation/NavigationController.kt

```diff
@@ -31,11 +32,13 @@ import kotlinx.coroutines.InternalCoroutinesApi
 @ExperimentalCoroutinesApi
 class NavigationController {
 
-    private val context: Context
     private val navigator: MapboxNativeNavigator
     private val locationEngine: LocationEngine
     private val locationEngineRequest: LocationEngineRequest
+    private val navigationOffboardRoute: NavigationOffboardRoute
```

Comment:
This is not needed / used 👀 https://github.com/mapbox/mapbox-navigation-android/pull/2332#discussion_r361965898